.github/workflows/validate-all.yml

```
@@ -19,10 +19,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Set up Python 3.8
+      - name: Set up Python '3.10'
         uses: actions/setup-python@v2
```

actionlint (1.7.7)

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)